sycl/doc/SYCLEnvironmentVariables.md

```diff
@@ -4,11 +4,23 @@ This file describes environment variables that are having effect on SYCL compile
 
 # Controlling SYCL RT
 
-| Environment variable | Description |
-| ----------- | ----------- |
-| SYCL_PI_TRACE | If set forces tracing of PI calls to stdout. |
-| SYCL_BE={PI_OPENCL,PI_OTHER} | When SYCL RT is built with PI this controls which plugin to use. |
-| SYCL_PRINT_EXECUTION_GRAPH | Print execution graph to DOT text file. Options are described below. |
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| SYCL_PI_TRACE | Any(*) | Force tracing of PI calls to stderr. |
+| SYCL_BE | PI_OPENCL, PI_OTHER | When SYCL RT is built with PI this controls which plugin to use. Default value is PI_OPENCL. |
+| SYCL_DEVICE_TYPE | One of: CPU, GPU, ACC, HOST | Force SYCL to use the specified device type. If unset, default selection rules are applied. If set to any unlisted value, this control has no effect. If the requested device type is not found, a `cl::sycl::runtime_error` exception is thrown. If a non-default device selector is used, a device must satisfy both the selector and this control to be chosen. This control only has effect on devices created with a selector. |
+| SYCL_PROGRAM_BUILD_OPTIONS | String of valid OpenCL build options | Override build options for all programs. |
+| SYCL_USE_KERNEL_SPV | Path to the SPIR-V binary | Load device image from the specified file. If runtime is unable to read the file, `cl::sycl::runtime_error` exception is thrown.|
+| SYCL_DUMP_IMAGES | Any(*) | Dump device image binaries to file. Control has no effect if SYCL_USE_KERNEL_SPV is set. |
+| SYCL_PRINT_EXECUTION_GRAPH | Described [below](#sycl_print_execution_graph-options) | Print execution graph to DOT text file. |
+
+`(*) Note: Any means this environment variable is effective when set to any non-null value.`
+
+**Warning:** the aforementioned environment variables are used for development
+and debugging of SYCL runtime and compiler. Their semantics are subject to change.
+Do not rely on these variables in production code.
```

Comment:
I think this warnings should be moved at the beginning of the section (or the whole document).